index.js

```diff
@@ -5,9 +5,9 @@ const oxmlContentTypes = toBytes('[Content_Types].xml');
 const oxmlRels = toBytes('_rels/.rels');
 
 module.exports = input => {
-	const buf = new Uint8Array(input);
```

Comment:
This is intentional, for browser support.

Reply:
this make new buffer it It takes performance.
Why an array of numbers is not good for checking?